### ISSUE TITLE
Custom RPC 

### ISSUE THREAD

**Issue by witchkingdom** [gemworks/gem-farm#91]
Hi, may i know if there is a way to set and use my own custom RPC ?

**Comment by StefosGeo** [comment]
Yes, set up an enviromental variable called VUE_APP_MAINNET_URL equals to the http endpoint of your cluster.

You can check out at src/composables/clusters.ts

**Comment by witchkingdom** [comment]
thank you so much for your help
